pkg/controller/subnet.go

```diff
@@ -2110,10 +2121,20 @@ func (c *Controller) calcSubnetStatusIP(subnet *kubeovnv1.Subnet) (*kubeovnv1.Su
 		klog.Error(err)
 		return nil, err
 	}
+	usingIPNums := len(podUsedIPs)
+	for _, podUsedIP := range podUsedIPs {
+		for _, excludeIP := range subnet.Spec.ExcludeIps {
+			if util.ContainsIPs(excludeIP, podUsedIP.Spec.V4IPAddress) || util.ContainsIPs(excludeIP, podUsedIP.Spec.V6IPAddress) {
+				// This ip cr is allocated from subnet.spec.excludeIPs, do not count it as usingIPNums
+				usingIPNums--
```

Comment:
need break here to skip remaining excludeIP

Reply:
fixed